tests/ui/include-macros/mismatched-types.stderr

```diff
@@ -1,8 +1,11 @@
 error[E0308]: mismatched types
-  --> $DIR/mismatched-types.rs:2:20
+  --> $DIR/file.txt:0:1
+   |
+   |
+  ::: $DIR/mismatched-types.rs:2:12
    |
 LL |     let b: &[u8] = include_str!("file.txt");
-   |            -----   ^^^^^^^^^^^^^^^^^^^^^^^^ expected `&[u8]`, found `&str`
+   |            -----   ------------------------ in this macro invocation
```

Comment:
I was a little nervous when I saw this that we were no longer printing the expected+actual types in the new output, but I expanded the diff and now see that the info is still present via a note that was in the previous output. Its a little more awkward to force people to look at the note to see this info, but its acceptable.